pkg/applicationmanager/controllers/applicationregistration.go

```diff
@@ -0,0 +1,215 @@
+package controllers
+
+import (
+	stork_api "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s/stork"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// IBMApp registration name
+	IBMApp = "ibm-app-reg"
+	// CouchBaseApp registration name
+	CouchBaseApp = "couchbase-app-reg"
+	// RedisClusterApp registration name
+	RedisClusterApp = "redis-app-reg"
+	// CassandraApp registration name
+	CassandraApp = "cassandra-app-reg"
+)
+
+func getSupportedCRD() map[string][]stork_api.ApplicationResource {
+	// supported CRD registration
+	defCRD := make(map[string][]stork_api.ApplicationResource)
+	// IBM CRD's
+	defCRD[IBMApp] = []stork_api.ApplicationResource{
+		{
+			GroupVersionKind: metav1.GroupVersionKind{
+				Kind:    "IBPCA",
+				Group:   "ibpcas.ibp.com",
+				Version: "ibp.com/v1alpha1",
```

Comment:
```suggestion
				Kind:    "IBPCA",
				Group:   "ibp.com",
				Version: "v1alpha1",
```
As dicsussed this needs to be updated